### PR TITLE
add java launcher example

### PR DESCRIPTION
**Pull request by malafeev**

### CODE REVIEW

File: java/launcher/src/main/java/com/lightstep/launcher/client/Client.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    if (targetURL == null || targetURL.length() == 0) {
+      targetURL = "http://127.0.0.1:8084";
+    }
+
+    final String spanEndpoint = System.getenv("OTEL_EXPORTER_OTLP_SPAN_ENDPOINT");
+    final String lsToken = System.getenv("LS_ACCESS_TOKEN");
+    final String serviceName = System.getenv("LS_SERVICE_NAME");
+    final String serviceVersion = System.getenv("LS_SERVICE_VERSION");
+
+    OtlpGrpcSpanExporter exporter = OpenTelemetryConfiguration.newBuilder()
```
Comment (kayousterhout) [inline java/launcher/src/main/java/com/lightstep/launcher/client/Client.java:30]: why do you have to do this manually (doesn't the launcher pull from the environment variables?)

File: java/launcher/src/main/java/com/lightstep/launcher/server/ExampleServer.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+public class ExampleServer {
+
+  public static void main(String[] args) throws Exception {
+    final String spanEndpoint = System.getenv("OTEL_EXPORTER_OTLP_SPAN_ENDPOINT");
+    final String lsToken = System.getenv("LS_ACCESS_TOKEN");
+    final String serviceName = System.getenv("LS_SERVICE_NAME");
+    final String serviceVersion = System.getenv("LS_SERVICE_VERSION");
+
+    OtlpGrpcSpanExporter exporter = OpenTelemetryConfiguration.newBuilder()
```
Comment (kayousterhout) [inline java/launcher/src/main/java/com/lightstep/launcher/server/ExampleServer.java:23]: same question as above re: why you have to set these?

**Review by kayousterhout — CHANGES_REQUESTED** [review]

**Comment by kayousterhout** [comment]
Also can you update the readme: https://github.com/lightstep/opentelemetry-examples#with-docker-compose

File: java/launcher/src/main/java/com/lightstep/launcher/client/Client.java (hunk 1 above)
Comment (malafeev) [reply java/launcher/src/main/java/com/lightstep/launcher/client/Client.java:25]: was added to show explicit configuration.
removed

File: java/launcher/src/main/java/com/lightstep/launcher/server/ExampleServer.java (hunk 2 above)
Comment (malafeev) [reply java/launcher/src/main/java/com/lightstep/launcher/server/ExampleServer.java:18]: removed

**Comment by malafeev** [comment]
readme is updated

**Review by kayousterhout — APPROVED** [review]

**Pull request merged**